README.md

```diff
@@ -1,10 +1,31 @@
-# Maximum Entropy Biasing of Epidomiology Models
+# Compartmental Epidomiology Modeling
 
-This python module implements [maximum entropy biasing](https://github.com/ur-whitelab/maxent) to modify epidomiology parameters to find the best fit to disease trajectory given a set of observations. These observations are time-averaged fractional values that can come from different compartments (S, E, A, I and R) of a known synthetic reference trajectory or real pandemic spread data.  
+``py0`` is a python implementation of compartmental disease modeling.
+
+![](docs/source/img/py_0.gif)
 
 ## Installation
 
-The package uses Keras (Tensorflow), geopandas and networkx.
+This package uses geopandas and networkx. Make sure you have ``gdal-config`` defined in your system before installing ``py0``. Assuming you are in a linux env, to install ``gdal`` run:
+```sh
+sudo apt-add-repository ppa:ubuntugis/ubuntugis-unstable
```

Comment:
Still think you should make this an optional dependency, like we do for things in hoomd-tf. People will be more likely to try if there are fewer install. Also, these instructions are not for "linux", these are for debian env. Will not work on centos, redhat, arch etc.

Reply:
will remove geopandas as a dependency.